docs/TektonPipeline.md

```diff
@@ -184,6 +185,7 @@ spec:
   performance:
     disable-ha: false
     buckets: 1
+    replicas: 1
```

Comment:
If i understand correctly only for Pipeline `replicas` field added here and for other components  `replicas` will be added as part of optional one right ?

Reply:
Yes, you are correct. we already have group of fields related to performance. So I added `replica` along with this only to pipeline controller